src/main/java/ui/listpanel/ListPanel.java

```
@@ -492,6 +564,46 @@ private void markAsRead() {
         }
     }
 
+    private void createNewWatchListMenuItem() {
+        Optional<GuiElement> item = listView.getSelectedItem();
+        if (!item.isPresent()) {
+            return;
+        }
```

Shouldn't this have some error handling rather than failing silently?